### ISSUE TITLE
Error centering multiple datasets at once

### ISSUE THREAD

**Issue by mkshaw** [bkeller2/mlmpower#6]
Hi Brian,

I'm trying to use the `center()` function to centre multiple datasets at once, but am getting the error 

Error:
✖ Unrecognized centering: `..1`
ℹ You must specify `cgm`, `cwc`, or `none`
Run `rlang::last_trace()` to see where the error occurred.

Here's an example of the code:

```
model <- (
    effect_size(
      icc = 0.25,
      within = 0.1,
      between = 0.1,
      product = 0.01,
      random_slope = 0.03
    )
    + outcome('y')
    + within_predictor('x1', icc = 0)
    + between_predictor('z1')
    + product('x1', 'z1') 
    + random_slope('x1')
  )

dat <- generate(model, n_within = 5, n_between = 50, ndata = 5) |> center(x1 = cwc)
```

I don't get that error with `ndata = 1`. Any ideas how to fix that?

**Comment by bkeller2** [comment]
@mkshaw 

It is a bug when trying to pass arguments from `...` down to other functions.

I have fixed this in 3ebc121414ad10a695d14310991f026ed71941cb and have submitted the fix to CRAN in `1.0.7`.

You can update it in the meantime using 

```r
remotes::install_github('bkeller2/mlmpower', build_vignettes = TRUE)
```

Alternatively, you would have to hardcode it:

```r
model <- (
    effect_size(
      icc = 0.25,
      within = 0.1,
      between = 0.1,
      product = 0.01,
      random_slope = 0.03
    )
    + outcome('y')
    + within_predictor('x1', icc = 0)
    + between_predictor('z1')
    + product('x1', 'z1') 
    + random_slope('x1')
  )

dat <- generate(model, n_within = 5, n_between = 50, ndata = 5)
dat_center <- lapply(dat, \(x) x |> center(x1 = cwc))
```

Thanks for finding this!

**Comment by mkshaw** [comment]
Thanks very much!
